GA_core/ga_classes.py

```diff
@@ -5,26 +5,51 @@ def update(self, generation, best_score):
 
 
 class Genetic_Algorithm:
+    """
+    gene = policy (per SKU)
+
+    chromosome = individual solution (e.g., for 100 SKUs, max 100 policies)
+
+    population = a number of chromosomes

```

Comment:
correct, a population is a collection of solutions. The cardinality of it is a parameter. You can have 100, 10000, 23452345, you name it.

The idea of a GA is that populations are evolving, leading to better scores solutions scores. By evolving we mean,
- combining/mating (crossovers), 
- mutating (random variation). Mutation can happen either on the population, or the chrom. or both. This is a parameter to allow the algorithm to get unstack in case it gets trapped in a local minima.